hack/robot-e2e/ansible.cfg

```
@@ -0,0 +1,7 @@
+[defaults]
+inventory = ${PWD}/inventory.yml
+forks = 10
+host_key_checking = False
+
+[ssh_connection]
+# pipelining = True
```

```suggestion
pipelining = True
```

This can speed up the ansible part. Unless it fails hard, I would enable it. Or completely remove it.